chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -552,7 +550,7 @@ public boolean RootsCubic(double[] coef, double[] roots) {
      * @param r see above
      * @return the computed result
      */
-    public double Voigt(double xx, double sigma, double lg, int r) {
+    public double voigt(double xx, double sigma, double lg, int r) {
```

Comment:
Codacy found an issue: [The method 'voigt(double, double, double, int)' has an NPath complexity of 2400, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)